static/src/javascripts/projects/commercial/modules/user-features.spec.js

```diff
@@ -143,7 +156,7 @@ describe('The isPayingMember getter', () => {
     it('Is false when the user is logged out', () => {
         jest.resetAllMocks();
         isUserLoggedIn.mockReturnValue(false);
-        expect(isAdFreeUser()).toBe(false);
+        expect(isPayingMember()).toBe(false);
```

Comment:
@gidsg I think you were the last to work on this test. I assume this is what you'd intended?